siphon/cdmr/coveragedataset.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2016 University Corporation for Atmospheric Research/Unidata.
 # Distributed under the terms of the MIT License.
 # SPDX-License-Identifier: MIT
+"""Provide access to TDS' Coverage Dataset."""
```

Comment:
TDS' took me a couple of glances. Maybe this reads cleaner? Provide access to a THREEDS server's coverage dataset